numpy/f2py/cfuncs.py

```diff
@@ -542,7 +542,7 @@
     'ARRSIZE'] = '#define ARRSIZE(dims,rank) (_PyArray_multiply_list(dims,rank))'
 cppmacros['OLDPYNUM'] = """\
 #ifdef OLDPYNUM
-#error You need to install NumPy version 13 or higher. See https://scipy.org/install.html
+#error You need to install NumPy version 0.13 or higher. See https://scipy.org/install.html
```

Comment:
This was left over from a [comment](https://github.com/numpy/numpy/pull/14231#discussion_r312112493) on gh-14231